### PR TITLE
Minimal templates banner

### LINKED ISSUE #201: Banner that takes users to Templates
The idea is to create a Banner that announces the Minimal Templates that are free of use and link to the Templates page

This banner should be located in the homepage

Maybe a design like this Hero would work

Just flip the image to the left side and the text to the right side

Draw attention that it is a Free Resource and the CTA should go to docs/templates

Should be a bit smaller in height to differentiate from the main hero

Should be below components and above templates in homepage

image

### PR DESCRIPTION
**Pull request by ChcJohnie**
closes #201 

- tried to create more space with shorter headlines (font sizes would require changing reusable FeaturePreview component, so Im keeping that as next option)
- more space in images added
- 404 keep being strange, for unknown reason started to work even before fix (but not locally) but found out that Link(s) to redirected page might cause trouble so changed CTA href to /docs/templates/minimal

### CODE REVIEW

File: src/app/(with-navbar)/page.tsx [hunk 1]
```diff
@@ -22,23 +21,8 @@ const Home = () => (
...
-            quality={100}
-            className="relative left-1/2 w-full min-w-[540px] -translate-x-1/2 px-4"
-          />
-          <div className="absolute inset-0 flex items-center justify-center bg-[linear-gradient(180deg,rgba(0,0,0,0.3)_0%,rgba(0,0,0,0.65)_50.1%,#000000_100%)]">
-            <div className="bg-pink-to-purple-gradient bg-clip-text text-3xl font-bold uppercase text-transparent drop-shadow-lg sm:text-6xl">
-              Coming soon
-            </div>
-          </div>
-        </div>
+      <FeaturePreview title="Choose yours templates" previewType="templates">
```
Comment (iamhectorsosa) [inline src/app/(with-navbar)/page.tsx:24]:
```suggestion
      <FeaturePreview title="Choose your templates" previewType="templates">
```

**Review by iamhectorsosa — COMMENTED** [review]
Looking good!

**Comment by iamhectorsosa** [comment]
Don't want to slow us down here but these components could be more generic if they're abstracted as components. They should take props and be re-usable otherwise it doesn't make much sense to abstract them. I'll leave it up to you, I don't want to slow us down like I said.

**Comment by ChcJohnie** [comment]
You mean hero block and badge I guess? 
I will see if I can add something tomorrow, otherwise I would merge and create low-prio refactor issue, ok?

**Comment by iamhectorsosa** [comment]
Sounds good

**Review by iamhectorsosa — APPROVED** [review]

**Pull request merged**